Assets/Scripts/Models/Tile.cs

```diff
@@ -41,6 +41,7 @@ public Tile(int x, int y, int z)
         Characters = new List<Character>();
         MovementModifier = 1;
         Utilities = new Dictionary<string, Utility>();
+        ReservedAsWorkSpotBy = new HashSet<Furniture>();
```

Comment:
I believe you are thinking of ReservedAsWorkSpot, which simply returns true if there is anything in ReservedAsWorkSpotBy (count > 0). A very simple check, but parses better than checking the count being greater than 0 from outside.

ReservedAsWorkSpotBy could perhaps use a lest linguistically awkward name, but I couldn't think of one.